src/views/Players/Players.test.js

```diff
@@ -0,0 +1,7 @@
+import { render } from '@testing-library/react';
+import Players from './Players';
+
+test('renders all players', () => {
+  const container = render(<Players />);
```

Comment:
Making this test async and awaiting with findByText would allow the Component's useEffect to run and render all of the data.